### DISCUSSION TITLE
Change css in single page component

### DISCUSSION THREAD

**Discussion by wontone18** [livewire/livewire#5621] [1. Help]
Hi,

I have a single page component with multiple components that follow different steps. I'm currently using Vite to load the CSS, but I'm looking for suggestions on how to dynamically change the CSS based on the component being loaded inside the main component.

I've tried several approaches, but the main CSS from component 1 always displays. When moving to component 2, I want its CSS to replace that of component 1, effectively disabling the CSS for component 1. It's important to note that both components are running inside the page component.

This is my page component display property change the component. I can combine my .scss file into one but i want to know any idea without combining this. 

 ```
@if($display == "detail") 
    @livewire('frontend.page.sharedbooking.detail',['shared'=>$shared])
    @livewire('frontend.page.sharedbooking.partial.about',['shared_about'=>$shared_booking->description])
    @livewire('frontend.page.sharedbooking.partial.faq')
    @livewire('frontend.page.sharedbooking.partial.meeting-policy')
    @livewire('frontend.page.sharedbooking.partial.otherboats',['shared'=>$shared_booking])
    @elseif($display == "customer")
    @livewire('frontend.page.sharedbooking.customer',['shared'=>$shared,'formdata' => $booking_form_data])
    @elseif($display == "payment")
    @livewire('frontend.page.sharedbooking.payment')
    @endif
```

**Comment by joshhanley** [comment]
@wontone18 I don't think Livewire supports updating css like that. You'd be better off using something like turbolinks/ turbo to swap the components as it should handle scripts and css for you too.

Hope this helps!
